src/test/java/com/google/javascript/gents/test_externs_map.json

```diff
@@ -0,0 +1,4 @@
+{
+  "jQuery": "JQuery"
```

Comment:
Yeah, I think there are so many we'd need an extension point anyway. I was planning on adding a canonical mapping file to google3 and have convert_js_to_ts use it; but if you prefer  I could add the default file in gents itself.